packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.integration.test.tsx

```diff
@@ -28,32 +59,30 @@ const getQuizHandler = rest.get(
   },
 )
```

Comment:
Is `getQuizHandler` the only MSW handler this test relies on? If so, can you convert it to a Jest mock, remove the MSW setup/teardown, and remove the `integration` part of the file name?